mobidam-sst-management-integration/pom.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The MIT License
+     Copyright © 2023 Landeshauptstadt München | it@M
+
+     Permission is hereby granted, free of charge, to any person obtaining a copy
+     of this software and associated documentation files (the "Software"), to deal
+     in the Software without restriction, including without limitation the rights
+     to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+     copies of the Software, and to permit persons to whom the Software is
+     furnished to do so, subject to the following conditions:
+
+     The above copyright notice and this permission notice shall be included in
+     all copies or substantial portions of the Software.
+
+     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+     IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+     FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+     AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+     LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+     THE SOFTWARE.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>de.muenchen.oss.mobidam</groupId>
+    <artifactId>mobidam-sst-management-integration</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>mobidam-sst-management-integration</name>
+    <description>spring-boot-starter to integrate mobidam-sst-management client into other components</description>
+    <url>https://github.com/it-at-m/mobidam-sst-management/</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+        </license>
+    </licenses>
+
+    <organization>
+        <name>Landeshauptstadt München - it@M</name>
+        <url>https://muenchen.digital/it-at-m/</url>
+    </organization>
+
+    <developers>
+        <developer>
+            <name>martind260</name>
+            <organization>Landeshauptstadt München</organization>
+            <url>https://github.com/martind260</url>
+            <roles>
+                <role>mobidam_admin</role>
+                <role>mobidam_devs</role>
+            </roles>
+        </developer>
+    </developers>
+
+    <distributionManagement>
+        <!-- Central Repository -->
+        <snapshotRepository>
+            <id>ossrh</id>
```

Comment:
Ich glaube die id muss mit der id aus dem release-Workflow übereinstimmen. Also entweder beides `central` oder beides `ossrh`

Reply:
@mirrodi Nachdem wir aber gar keine Snapshots bauen, brauchen wir diese Passage wohl gar nicht. Es steht aber in der Doku als "Muss" drin. Weißt Du dazu mehr?

Reply:
ne, leider nicht, aber laut der kurzen Recherche würde ich fast sagen, dass man das nicht braucht